extensions/youtube-downloader/package.json

```diff
@@ -52,6 +52,7 @@
     }
   ],
   "dependencies": {
+    "@distube/ytdl-core": "^4.13.5",
```

Comment:
I think you can remove the "ytdl-core" dependency then (a few lines below, but I cannot comment on unchanged lines)

Reply:
Removed.

Reply:
Also, you should be able to delete the `patches` folder and the `patch-package` library along with the `postinstall` script.

Thanks for the contribution and sorry for the delay in responding I was out last week.

Reply:
Done.